### DISCUSSION TITLE
problem with folding in dashboard

### DISCUSSION THREAD

**Discussion by Tornado300** [folke/snacks.nvim#220] [Q&A] [answered]
hey I'm quite new to snacks and i have a problem:
I'm using the indent folding method in neovim, but when i open the dashboard it automatically folds it...

i tried to fix it with an autocmd:
```lua
vim.api.nvim_create_autocmd({"FileType"}, {
  pattern = {"snacks_dashboard"},
  callback = function()
    vim.opt_local.foldmethod = "manual"
    vim.cmd("normal! zR") -- Open all folds
  end,
})
```
but that didn't work. How can i fix this problem or is it not possible ?

Thanks in advance

**Comment by folke** [comment, marked as answer]
`opts.styles.dashboard.wo.foldmethod = "manual"`
For now, set the option above. I already fixed it on a different branch.

**Reply by Tornado300** [reply]
Thanks for the fast reply, but where do i need to put it ? :sweat_smile:

**Reply by folke** [reply]
In your lazy spec. See the install instructions.

**Reply by Tornado300** [reply]
Thanks, just found it myself XD

btw, awesome projects <3
